content/4_Gold/Intro_DP.mdx

```diff
@@ -97,7 +94,7 @@ against.
 
 ## Classical Problems
 
-The next few modules provide examples of some classical problems, or Dynamic
+The next few modules provide examples of some classical problems: Dynamic
```

Comment:
A colon doesn't really make sense here. Maybe use brackets instead?

Reply:
why not?